fission/src/Synthesis.tsx

```diff
@@ -95,6 +95,11 @@ function Synthesis() {
                 .catch(_ => LoadMirabufRemote(DEFAULT_MIRA_PATH))
                 .catch(console.error)
 
+			const miraAssembly = await LoadMirabufRemote(mira_path, MiraType.ROBOT)
+				.catch(
+					_ => LoadMirabufRemote(DEFAULT_MIRA_PATH, MiraType.ROBOT)
+				).catch(console.error);
```

Comment:
There seems to be a left over merge conflict here. Duplicate definition of `miraAssembly`.